src/services/config.ts

```diff
@@ -0,0 +1,9 @@
+import { ConfigurationParser } from '@aerogear/core';
+
+declare var require: any
+
+var appConfig = require('../mobile-services.json');
+
+// TODO config part is too verbose.
+// We ideally looking for helper method instead of class.
+export let config = new ConfigurationParser(appConfig);
```

Comment:
:+1: maybe parseConfig('mobile-services.json')

Reply:
getConfig could be more descriptive.

Reply:
Would it be possible to import the JSON from ConfigParser instead? Since mobile-config.json location is by default, the user would only need to do a `new ConfigParser()` and it would throw an error if the JSON is not there. Just like Android with google-services.